src/core/lib/attribute-transition-model.js

```diff
@@ -1,13 +1,11 @@
-import {GL, Model, Geometry} from 'luma.gl';
-
 const ATTRIBUTE_MAPPING = {
   1: 'float',
   2: 'vec2',
   3: 'vec3',
   4: 'vec4'
 };
 
-function getShaders(transitions) {
+export function getShaders(transitions) {
```

Comment:
I will rename this file to `attribute-transition-utils.js` after initial review, kept it as is for easier review.